pkg/testutil/setup.go

```diff
@@ -39,9 +39,9 @@ func SetupTestingEnv(params *SetupTestingEnvParams) (logging.Logger, api.ClientW
 	viper.SetDefault("setup_lakefs_timeout", defaultSetupTimeout)
 	viper.SetDefault("endpoint_url", "http://localhost:8000")
 	viper.SetDefault("s3_endpoint", "s3.local.lakefs.io:8000")
-	viper.SetDefault("access_key_id", "")
-	viper.SetDefault("secret_access_key", "")
-	viper.SetDefault("storage_namespace", fmt.Sprintf("s3://%s/%s", params.StorageNS, xid.New().String()))
+	viper.SetDefault("access_key_id", params.AdminAccessKeyID)
+	viper.SetDefault("secret_access_key", params.AdminSecretAccessKey)
+	viper.SetDefault("storage_namespace", fmt.Sprintf("s3://%s", params.StorageNS))
```

Comment:
Did you remove the prefix by accident?

Reply:
No, I'm adding the prefix later during creation of the repository (generateUniqueStorageNamespace)